src/Moltin/Cart/Storage/LaravelFile.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+/**
+ * This file is part of Moltin Cart, a PHP package to handle
+ * your shopping basket.
+ *
+ * @package moltin/cart
+ * @author Alan Wynn <hello@alanwynn.me>
+ * @link http://github.com/moltin/cart
+ *
+ */
+
+namespace Moltin\Cart\Storage;
+
+use Moltin\Cart\Item;
+use Moltin\Cart\Storage;
+use Illuminate\Support\Facades\Config;
+
+class LaravelFile implements \Moltin\Cart\StorageInterface
+{
+    protected $storagePath;
+
+    protected $identifier;
+
+    protected static $cart = array();
+
+    public function __construct()
+    {
+        $this->storagePath = storage_path('Cart');
```

Comment:
I know this is an Laravel4 function, is it available in L5? And, Is it worth making the name 'Cart' configurable in some way?